app/main.py

```diff
@@ -14,6 +14,25 @@
     "John": 4,
     "Sergio": 3,
 }
-collection_of_coins = {1, 2, 25}
 
-# write your code here
+# Изменяем collection_of_coins на frozenset
+collection_of_coins = frozenset([1, 2, 25])  # Изменяем на frozenset
+
+# Определяем словарь для хранения всех переменных, сгруппированных по типу
+sorted_variables = {
+    "mutable": [
+        my_favourite_films,
+        marks,
+        collection_of_coins,  # Теперь это frozenset, но может остаться в списке mutable
```

Comment:
The `collection_of_coins` is a `frozenset`, which is immutable. It should be moved to the 'immutable' list instead of 'mutable'.